### ISSUE TITLE
Sort on save not working for edge cases

### ISSUE THREAD

**Issue by essarn** [windicss/windicss-intellisense#224]
Might have something to do with the released feature in #161

Consider this code where `font-custom` is a custom font family defined in the config.
```html
<a-col class="font-custom text-15px font-bold leading-20px">
    Hello World
</a-col>
```
When sorting this code (either via on save or manually) is this the result.
```html
<a-col class="font-custom font-bold font-bold leading-20px">
    Hello World
</a-col>
```

The font size has been replaced incorrectly and there are now duplicate classes present. 

Please not that if the classes are recorded might some other combination or duplication of classes happen. In other words, the sorting is not consistent between runs. 

Extension version: `v0.20.1`

EDIT: It looks like the problem happens when a class name that is last in the list needs to be moved.

**Comment by alexanderniebuhr** [comment]
thanks this is a regression. I just reverted the release with [v0.20.2](https://github.com/windicss/windicss-intellisense/releases/tag/v0.20.2)

**Comment by IgnisDa** [comment]
Yes sorry for the faulty PR. I think this will happen anytime a custom class is used. Will try to clean up my PR asap.

**Comment by IgnisDa** [comment]
@essarn Could you send me the windiconfig you used for this?

**Comment by alexanderniebuhr** [comment]
@IgnisDa for my test project I use, where it also breaks:

```js
const { defineConfig } = require('windicss/helpers')

module.exports = defineConfig({
  darkMode: 'media',
  theme: {
    extend: {
      colors: {
        primary: {
          DEFAULT: ({ opacityVariable, opacityValue }) => {
            if (opacityValue !== undefined) {
              return `rgba(var(--primary-dynamic), ${opacityValue})`
            }
            if (opacityVariable !== undefined) {
              return `rgba(var(--primary-dynamic), var(${opacityVariable}, 1))`
            }
            return `rgb(var(--primary-dynamic))`
          },
          50: 'var(--primary-50)',
          100: 'var(--primary-100)',
          200: 'var(--primary-200)',
          300: 'var(--primary-300)',
          400: 'var(--primary-400)',
          500: 'var(--primary-500)',
          600: 'var(--primary-600)',
          700: 'var(--primary-700)',
          800: 'var(--primary-800)',
          900: 'var(--primary-900)',
        },
      },
      cursor: {
        'ns-resize': 'ns-resize',
        'col-resize': 'col-resize',
        'n-resize': 'n-resize',
        's-resize': 's-resize',
      },
      lineClamp: {
        sm: '2',
      },
    },
  },
  plugins: [
    require('windicss/plugin/forms'),
    require('windicss/plugin/line-clamp'),
    require('windicss/plugin/typography'),
  ],
  shortcuts: {
    nrmlz: {
      '-webkit-text-size-adjust': '100%',
      'font-family': `ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"`,
      'line-height': '1.5',
      margin: '0',
    },
    fit: {
      width: 'max-content',
    },
  },
})
```

**Comment by alexanderniebuhr** [comment]
should be fixed in 0.21.0
